### DISCUSSION TITLE
Adding Commands

### DISCUSSION THREAD

**Discussion by galaxea** [Dadangdut33/Speech-Translate#8] [Ideas]
This is so useful and I appreciate the hard work. Is there a way to add commands such as suppress tokens that can be utilized when running a transcription file? I like to set it at 0 so it understands commands such as 'open quote' when running it through CMD.

**Comment by Dadangdut33** [comment]
I don't quite understand what you mean. Can you give me a more thorough example or explanation

**Reply by galaxea** [reply]
For my usage of dictating fiction rough drafts, I use the suppress tokens command so when I utilize verbal open quote and close commands, it transcribes with " symbols. So the example of how I use it is, whisper audio.mp3 --model small.en --suppress_tokens 0

I am just wondering if there's a way to add commands that are in the -help command to the end of transcribing, if they are needed to be used by someone.

**Reply by Dadangdut33** [reply]
oooh i get what you mean now. This is a good idea i should probably add all the parameter options to the settings
